tests/dat/actions/concurrent.js

```diff
@@ -1,6 +1,8 @@
 // Licensed to the Apache Software Foundation (ASF) under one or more contributor
 // license agreements; and to You under the Apache License, Version 2.0.
 
+const Promise = require('bluebird');
```

Comment:
what is this for?

Reply:
I wanted to use `finally` to decrement the counter again. But finally is only part of node10. The alternative on handling the counter would be to duplicate the code to decrement it (once in error case and once in success case).

That means I only had the choice between using Promises of `bluebird` in default node version or to use native Promises with node10. And I decided to use `bluebird` as it seems to be already part of our action container and I didn't want to hardcode the version for these tests.